app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/cloud_database.rb

```diff
@@ -1,2 +1,12 @@
 class ManageIQ::Providers::IbmCloud::VPC::CloudManager::CloudDatabase < ::CloudDatabase
+  include ProviderObjectMixin
```

Comment:
Probably easier if we add this to the core `CloudDatabase` model, otherwise we'll have to include this in every provider subclass

Reply:
https://github.com/ManageIQ/manageiq/pull/21810